lib/views/help/about.tr.html.erb

```diff
@@ -7,5 +7,10 @@
 
   <p>Bu site, <a href="https://www.tbmm.gov.tr/kanunlar/k4982.html">4982 Sayılı Bilgi Edinme Hakkı Kanunu</a>’nun etkin bir şekilde kullanımını desteklemek amacıyla, Bağımsız Gazetecilik Platformu Punto24 (P24) ve Londra merkezli Article19 tarafından kurulmuştur. Site üzerinden Türkiye’deki kamu kurum ve kuruluşlarına Bilgi Edinme başvuruları yapılabilmekte, başvurular ve gelen cevaplar kamuya açık bir şekilde internette görünür olmaktadır.</p>
 
+  <p>
+  <%= link_to "Gazeteciler için Bilgi Edinme Hakkı Kanunu Kılavuzu",
+              asset_path("/assets/4982.pdf") %>
```

Comment:
This doesn't look right.

[`asset_path`](https://api.rubyonrails.org/v4.2.10/classes/ActionView/Helpers/AssetUrlHelper.html#method-i-asset_path) shouldn't need the `"/assets/"` prefix.

Reply:
To test this, run the app in production mode (you'll need to compile assets) and probably hack https://github.com/mysociety/alaveteli/blob/develop/config/environments/production.rb#L24-L26 in.

Reply:
Works locally with suggested modification. Will fix that and deploy!

Reply:
On second thoughts, this will serve this downloadable file as e.g. http://localhost/assets/4982-06dd1ffa2dadf5fa20420cb866ee16b5.pdf which is ... not ideal.